styx-common/src/main/java/com/spotify/styx/storage/AggregateStorage.java

```diff
@@ -155,13 +156,18 @@ public void delete(WorkflowId workflowId) throws IOException {
   }
 
   @Override
-  public void updateNextNaturalTrigger(WorkflowId workflowId, Instant nextNaturalTrigger) throws IOException {
-    datastoreStorage.updateNextNaturalTrigger(workflowId, nextNaturalTrigger);
+  public void updateNextNaturalTrigger(WorkflowId workflowId, TriggerInstantSpec triggerSpec)
+      throws IOException {
+    datastoreStorage.updateNextNaturalTrigger(workflowId, triggerSpec);
   }
 
   @Override
-  public Map<Workflow, Optional<Instant>> workflowsWithNextNaturalTrigger()
-      throws IOException {
+  public void updateNextNaturalTriggerOld(WorkflowId workflowId, Instant instant) throws IOException {
```

Comment:
It's used from a test that simulates the storage condition before migrating to this. Will add a VisibleForTesting annotation.